code.py

```diff
@@ -53,12 +56,33 @@ def nextLine(self, pose):
 
 
 	# Function D in paper: returns dist b/w oldpose and next line
-	def nextDist(self, pose, line):
+	def nextDist(self, pose, line, index_bot):
+		x0 = pose[0]
+		x1 = pose[1]
```

Comment:
y0?

Reply:
Oh, I am sorry, I must've typed it wrong. rest assured , I did it correctly in my machine and the code is working